src/actions/assets/Potree.js

```diff
@@ -0,0 +1,53 @@
+import AssetActionPlugin from "../AssetActionPlugin";
+import URI from 'urijs';
+import i18n from "../../i18n";
+
+const POTREE_SUPPORTED_TYPES = [
+  'application/vnd.laszip+copc',
+];
+
+// this.component.filename.endsWith or this.asset.href.includes
+const POTREE_SUPPORTED_FILEEXTS = [
+ 'cloud.js', 'metadata.json', 'ept.json'
```

Comment:
cloud.js and metadata.json are very ambiguous, especially the latter is often used also for non-potree. I think it will lead to a lot of false positives, so not convinced about this approach.

Reply:
Are these used for items in a lot of existing STAC catalogs? `cloud.js` it the root file/access-point for a potree v1 dataset, and `metadata.json` is used for potree v2 tilesets. should these be checked differently?

Reply:
Not sure about cloud.js, but I have at least one examples in mind that use metadata.json for non-potree (CDSE STAC API). And it's pretty expectable as the name is very generic. Can we somehow limit this to when the pointcloud extension is implemented for example?